src/kvstore/Part.cpp

```diff
@@ -420,11 +424,47 @@ bool Part::preProcessLog(LogID logId, TermID termId, ClusterID clusterId, const
 }
 
 void Part::cleanup() {
-  LOG(INFO) << idStr_ << "Clean rocksdb commit key";
-  auto res = engine_->remove(NebulaKeyUtils::systemCommitKey(partId_));
-  if (res != nebula::cpp2::ErrorCode::SUCCEEDED) {
-    LOG(WARNING) << idStr_ << "Remove the committedLogId failed, error "
-                 << static_cast<int32_t>(res);
+  LOG(INFO) << idStr_ << "Clean rocksdb part data";
+  // Remove the vertex, edge, index, systemCommitKey, operation data under the part
+  const auto& vertexPre = NebulaKeyUtils::vertexPrefix(partId_);
+  auto ret = engine_->removeRange(NebulaKeyUtils::firstKey(vertexPre, vIdLen_),
+                                  NebulaKeyUtils::lastKey(vertexPre, vIdLen_));
+  if (ret != nebula::cpp2::ErrorCode::SUCCEEDED) {
+    LOG(ERROR) << idStr_ << "Remove the part vertex data failed, error "
```

Comment:
insert one space between idStr_ and Remove

Reply:
IIRC, idStr_ already has one space suffix.

Reply:
There is already a space at the end of `idStr_ `string, so there is no need to insert space.